build.sh

```diff
@@ -370,8 +370,10 @@ function processSource {
   perl .pre-process-tag-omission.pl < "$HTML_TEMP/source-expanded-2" | perl .pre-process-index-generator.pl > "$HTML_TEMP/source-whatwg-complete" # this one could be merged
 
   function runWattsi {
-    # Input arguments: $1 is the file to run wattsi on, $2 is a directory for wattsi to write output
-    # to
+    # Input arguments:
+    # - $1 is the file to run Wattsi on
+    # - $2 is a directory for Wattsi to write output to
+    # - $3, optional, is the syntax highlighter command to pass to Wattsi
```

Comment:
+    # - $3, optional, is the syntax highlighter command to pass to Wattsi 

More specifically, Wattsi actually needs to this to be a pathname to the command (script). That is, even if you put  the highlighter script into your `PATH` environment, Wattsi doesn’t read that.